02_bubble/nicholascross/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, bubble([]int{3, 2, 1, 5}))
+	fmt.Fprintln(out, insertion([]int{3, 2, 1, 5}))
+}
+
+func bubble(s []int) []int {
+	count := len(s)
+	sort := make([]int, count)
+	copy(sort, s)
+
+	for sorted := false; !sorted; {
+		sorted = true
+
+		for i := 0; i+1 < count; i++ {
```

Comment:
To be consistent with the insertion sort, you could start this at 1 and use `i-1` in the loop:
```
for i := 1; i < count; i++ {
...
}
```

This makes the loop conditional a little less cumbersome.

As this is related to general programming and not Go-specific, not fixing this will not prevent approval or merging.